warehouse/static/sass/blocks/_vertical-tabs.scss

```diff
@@ -54,12 +54,13 @@
   &__tab {
     display: block;
     padding: $spacing-unit / 2;
-    cursor: pointer;
 
-    &:hover,
-    &:focus {
-      text-decoration: none;
-      color: darken($primary-color, 10)
+    @at-root {
+      a#{&}:hover,
+      a#{&}:focus {
+        text-decoration: none;
+        color: darken($primary-color, 10)
+      }
```

Comment:
It's not clear to me what these changes are for, could you explain?

Reply:
Statistic items don't have a link in all cases (GitLab star view is not implemented yet). To make it clear `span` is used instead of `a`, but css was still changing span element on hover. The code above apply css only for `a`.

The code above is compiled to `a.<class>:hover, ...`

Reply:
Could we do this instead by simply removing this line: 
https://github.com/pypa/warehouse/blob/master/warehouse/static/sass/blocks/_vertical-tabs.scss#L57

?

Reply:
No, in that case hovering over statistic item without link - `span` would still get same hover styling as links. 

If you look at screenshot https://github.com/pypa/warehouse/pull/4384#issuecomment-408613727 hovering over `Stars` make no difference to its styling, without that addition, hovering over `Star` would change the text color to blue.